src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -372,56 +422,49 @@ def build(self, notification_uuid: str | None = None) -> Union[SlackBlock, Slack
                 fields=fields,
                 footer=footer,
                 text=text,
-                title=build_attachment_title(obj),
-                title_link=get_title_link(
-                    self.group,
-                    self.event,
-                    self.link_to_event,
-                    self.issue_details,
-                    self.notification,
-                    ExternalProviders.SLACK,
-                    rule_id,
-                    notification_uuid=notification_uuid,
-                ),
+                title=title,
+                title_link=title_link,
                 ts=get_timestamp(self.group, self.event) if not self.issue_details else None,
             )
 
         # build up the blocks for newer issue alert formatting #
-        tags = get_tags(event_for_tags, self.tags)
+
         # build title block
-        title_link = get_title_link(
-            self.group,
-            self.event,
-            self.link_to_event,
-            self.issue_details,
-            self.notification,
-            ExternalProviders.SLACK,
-            rule_id,
-            notification_uuid=notification_uuid,
-        )
-        blocks = [
-            self.get_markdown_block(
-                text=f"<{title_link}|*{escape_slack_text(build_attachment_title(obj))}*>  \n{text}",
-            )
-        ]
+        if text and not self.actions:
+            text = f"```{text.lstrip(' ')}```"
```

Comment:
what was the reason for putting this in a code block?

Reply:
I put the code formatting to match the designs like this:

<img width="606" alt="Screenshot 2024-01-23 at 10 44 24 AM" src="https://github.com/getsentry/sentry/assets/29959063/37573d1c-7a4a-4706-8b42-3f74820c6277">

though now that you point it out, it looks a little funny for the post-action notifications :/ ideally the error text would be code formatted, but the "Issue archived by colleen" wouldn't be, that's why I have the `not self.actions` check 🤔 maybe I can parse the string and get the desired effect?

and I added the lstrip the white space before a notification like this: 

<img width="440" alt="Screenshot 2024-01-23 at 10 43 20 AM" src="https://github.com/getsentry/sentry/assets/29959063/418d7664-6db8-4b04-97c2-c5d96e8347fe">

Reply:
and good callout for the statuses, I didn't even notice that because I only ever had "new" 😅

Reply:
ok I updated everything you pointed out!! working on tests now, but it looks a lot better.